spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/rdd/CassandraRDDSpec.scala

```diff
@@ -1107,6 +1108,23 @@ class CassandraRDDSpec extends SparkCassandraITFlatSpecBase {
       (10, 11, "1011"),
       (10, 12, "1012"))
 
+    // Try to delete rows older than year 2000.
+    sc.cassandraTable(ks, "delete_wide_rows1").where("key = 10")
+      .deleteFromCassandra(ks, "delete_wide_rows1",
+        writeConf = WriteConf(timestamp = TimestampOption.constant(new DateTime(2000, 1, 1, 7, 8, 8, 10))))
+
```

Comment:
What happens with a ttl on Deletes? Checking for myself now...

Reply:
not allowed. We should probabaly pre-throw an exception if the writeconf is set with a ttl on a delete operation? Or warn that it it is illegal and ignore it?

Reply:
ignore it